Lib/test/test_pdb.py

```diff
@@ -3948,6 +3948,13 @@ def _create_fake_frozen_module():
         # verify that pdb found the source of the "frozen" function
         self.assertIn('x = "Sentinel string for gh-93696"', stdout, "Sentinel statement not found")
 
+    def test_empty_file(self):
+        script = ''
+        commands = 'q\n'
+        # Anything reasonable is acceptable here, as long as it does not halt
+        _, _ = self.run_pdb_script(script, commands)
+        _, _ = self.run_pdb_module(script, commands)
```

Comment:
```suggestion
        self.run_pdb_script(script, commands)
        self.run_pdb_module(script, commands)
```

Reply:
I thought about this again and I think we should at least check for something so I changed the test a bit.